src/main/java/com/notably/model/suggestion/SuggestionModelImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.notably.model.suggestion;
+
+import java.util.List;
+import java.util.Optional;
+
+import javafx.beans.property.Property;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+/**
+ * The implementation class of SuggestionModel.
+ */
+public class SuggestionModelImpl implements SuggestionModel {
+    private ObservableList<SuggestionItem> suggestions;
+    private String commandInputText;
+    private Property<Optional<String>> commandTextProperty;
+
+    @Override
+    public ObservableList<SuggestionItem> getSuggestions() {
+        return suggestions;
+    }
+
+    @Override
+    public void setSuggestions(List<SuggestionItem> suggestions) {
+        this.suggestions = FXCollections.observableArrayList(suggestions);
```

Comment:
Don't do this! This won't trigger the subscriber about the update in the observable list, since you're changing the reference to the observable list.

Instead, call the existing `ObservableList`'s [setAll](https://docs.oracle.com/javase/8/javafx/api/javafx/collections/ObservableList.html#setAll-java.util.Collection-) method